Sources/DronecodeSDKSwiftDemo/Controllers/MapViewController.swift

```diff
@@ -147,6 +148,23 @@ class MapViewController: UIViewController, CLLocationManagerDelegate {
             .subscribe()
```

Comment:
I believe this could run it async:

```suggestion
            .subscribeOn(ConcurrentDispatchQueueScheduler(qos: .background))
            .subscribe()
```

Reply:
Thanks, that did it.